pkg/solver/matcher.go

```diff
@@ -18,58 +19,311 @@ func (a ListOfResourceOffers) Less(i, j int) bool {
 }
 func (a ListOfResourceOffers) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 
+type AllowlistItem struct {
+	Enabled  bool
+	ModuleID string
+	Version  string
+}
+
+func loadAllowlist(filepath string) ([]AllowlistItem, error) {
+		file, err := os.Open(filepath)
+		if err != nil {
+			return nil, err
+		}
+		defer file.Close()
+	
+		bytes, err := ioutil.ReadAll(file)
+		if err != nil {
+			return nil, err
+		}
+	
+		var allowlist []AllowlistItem
+		if err := json.Unmarshal(bytes, &allowlist); err != nil {
+			return nil, err
+		}
+	
+		return allowlist, nil
+	}
+
+	// strip out https- keep the org and module in place
+	// https://lilypad.tech/cowsay -> lilypad.tech/cowsay
```

Comment:
It is more like `https://github.com/Lilypad-Tech/lilypad-module-cowsay` -> `lilypad-tech/lilypad-module-cowsay` (take notice how they are all lower case)